articles/libraries/lock-ios/password-reset-ios.md

```diff
@@ -0,0 +1,52 @@
+---
+toc_title: Password Reset 
+description: All you need to know about password reset in iOS
+---
+
+# Lock iOS: Password Reset
+
+## Introduction
+You can allow the user to reset the password for any database connections. 
+We strongly suggest reading [this article](/connections/database/password-strength) before implementing this.
+
+If you use Lock UI you can hide or show a "Reset password" button by setting property `disableResetPassword`, which default value is set to `false`.
+If you implement your custom UI you need to send a password reset email to a user using `A0APIClient`.
+
+## Important considerations
+Please note that passwords can only be changed for users signing in using database connections. If a user is signing in with a social or enterprise connection, their password would need to be reset in those platforms.
+
+## Code snippets
+
+### Objective-C version
+
+```objective-c
```

Comment:
@lobo I can't see any highlighting https://auth0-docs-content-pr-1900.herokuapp.com/docs/libraries/lock-ios/password-reset-ios#objective-c-version I think the valid one is `objc` (not really sure but I've seen it used)